news/2018-12-28-the-qat-gazette-december-2018.md

```diff
@@ -0,0 +1,83 @@
+
+### Proposed changes to the QAT
+
+As mentioned several times in this article already, significant changes could be just on the horizon for the Quality Assurance Team, which may drastically change how we operate in the future. You can find more details on this announcement and the reasons for it in the Development forum, where [discussions towards multiple osu!team and community proposals](https://osu.ppy.sh/community/forums/topics/842601) are currently taking place.
+
+The first of these changes has been one of leadership, with [-Mo-](https://osu.ppy.sh/u/2202163) replacing myself as QAT Leader, alongside the continuing [Mao](https://osu.ppy.sh/users/2204515). Whilst I have thoroughly enjoyed my time at the head of the QAT, my free time available for osu! has been reduced considerably over the past few months, and I no longer feel I can properly fulfil the expectations of my role, or the community who elected me to it. On a positive note, new faces provide the potential for new direction, so please support -Mo- warmly as he settles into the challenge of management!
+ 
+One key aspect of these changes is that the base proposal was intentionally “vague”, in an effort to encourage community discussion and the generation of fresh ideas. When it comes to shaping the future of the osu! mapping and modding scene, there’s perhaps never been a better time than right now to make your voice heard! If you feel you have a great idea, or the drive to help others realise theirs, head on over to the above-mentioned proposals and get discussing!
+
+### Rules Regarding Mapping Commissions
+
+Just prior to the proposed Quality Assurance Team changes, significant discussion amongst the osu!team and QAT took place to clarify our position on paid-for mapping and modding contributions, as well as how we should handle such cases going forward. Whilst such mapping and modding commissions are certainly not a new appearance in osu!, they have gained significant traction and mainstream attention recently. Coupled with osu!’s awesome and ever-expanding library of licensed *Featured Artists*, it was decided to tidy up some potentially grey areas.
+ 
+You can read an [announcement from the osu!team](https://osu.ppy.sh/community/forums/topics/840838) on the topic for a detailed outlook, but in brief summary, commissions will be permitted to a certain extent, provided they are undertaken tastefully and the users profiting from them do not occupy a position within the Beatmap Nominators, Quality Assurance or other group within the osu!team. With this move, we wish to both emphasize osu!’s core “free-to-win” value, and promote a positive image and reputation with our current and potential future Featured Artists.
```

Comment:
or other group -> or other groups

Reply:
Either is fine, people can be in one group or multiple